### DISCUSSION TITLE
How can I apply both ventilation setback and disable air-mixing in Controller:OutdoorAir?

### DISCUSSION THREAD

**Discussion by helgakovacs** [NREL/EnergyPlus#10851] [Q&A]
Hi,

I am using DOAS air system in the building performance simulation with 23.1 EnergyPlus and I would like to only use fresh air as ventilation supply air (and do not mix it with the extracted air from the zone). I have found a way for that: by setting the Minimum Fraction of Outdoor Air Schedule to always 1.

However it will overwrite the ventilation schedule (Minimum Outdoor Air Schedule) I specified with setback during the night because of the following hierarchy:

_"There are various minimum and maximum limits on the outdoor air flow rate which are applied in the following order. Limits that fall later in the list can override earlier limits."_

- **OA flow rate = Minimum Outdoor Air Flow Rate * Minimum Outdoor Air Schedule**
- Apply economizer controls
- OA flow rate ≥ Controller:MechanicalVentilation OA flow rate
- OA flow rate ≥ System exhaust flow rate
- **(OA flow rate)/(Current mixedair flowrate) ≥ Minimum Fraction of Outdoor Air Schedule**

Source: InputOutputReference, Chapter 1.50.3

Could you please help what would be the way to use only fresh air in the ventilation system and also keep the schedule I specify with Minimum Outdoor Air Schedule and apply ventilation setback?

Kind regards,
Helga

**Comment by mjwitte** [comment]
The total system supply flow needs to be reduced to match the lower outdoor air flow rate. You should be able to do this using `AvailabilityManager:NightVentilation`.

Or, you can rip out the OA system, OA controller, and return path for the DOAS, so it is always 100% OA. See example file 5ZoneAirCooledWithDOASAirLoop. For this configuration, use `AirTerminal:SingleDuct:ConstantVolume:NoReheat` with a "Design Specification Outdoor Air Object Name" and use the schedule in the DesignSpecification:OutdoorAir object(s) to control the flow. It should also be possible to use  `AvailabilityManager:NightVentilation` with this configuration (but I can't say that I've actually tried that).

**Reply by helgakovacs** [reply]
Thank you for the reply.

Would the two solutions that you proposed work for winter season setback too? E.g. if it is an office building and the night ventilation flow rate is lower both in heating and cooling season. (From the documentation, it seems that AvailabilityManager:NightVentilation is to be used for night cooling in the summer.)

**Comment by mjwitte** [comment]
You are correct, AvailabilityManager:NightVentilation will not work for this. But with the existing system (with OA mixer and controller), you should be able to schedule the system supply flow rate to match the OA flow rate schedule by adding  "Design Specification Outdoor Air Object Name" to the DOAS terminal units.
